File: program/find-area-of-a-rectangle/FindAreaOfARectangle.kt
import java.util.Scanner

fun main() {
    val number1 = Scanner(System.`in`)
    println("Enter the Length: ")
    val length: Double = number1.nextDouble()

    val number2 = Scanner(System.`in`)
    println("Enter the breadth: ")
    val breadth: Double = number2.nextDouble()

    val Area = length * breadth
    println("The area of rectangle is $Area ")
}

File: program/find-area-of-a-rectangle/find_area_of_a_rectangle.go
package main

import "fmt"

func main() {
	var l, b float32
	fmt.Printf("Enter length of rectangle: ")
	fmt.Scanf("%v", &l)
	fmt.Printf("Enter breadth of rectangle: ")
	fmt.Scanf("%v", &b)

	fmt.Printf("Area of rectangle: %v\n", l*b)
}
